src/search/search-bar.test.tsx

```diff
@@ -351,17 +352,155 @@ describe('Search bar', () => {
 		expect(within(selector).getByText(app1SearchView.label)).toBeVisible();
 	});
 
-	it.todo(
-		'should show the module with lowest priority if both store and session storage are undefined'
-	);
+	it('should show the module with the lowest priority if both store and session storage are undefined', () => {
+		const app1 = generateCarbonioModule({ priority: 1 });
+		const app2 = generateCarbonioModule({ priority: 2 });
+		const app1Route = generateModuleRouteDescriptor({
+			label: app1.name,
+			position: 1,
+			id: app1.name,
+			route: app1.name
+		});
+		const app2Route = generateModuleRouteDescriptor({
+			label: app2.name,
+			position: 2,
+			id: app2.name,
+			route: app2.name
+		});
+		setupAppStore([app1, app2], [app1Route, app2Route]);
+		const app1SearchView = {
+			app: app1.name,
+			icon: app1.icon,
+			id: app1.name,
+			label: app1.display,
+			position: 6,
```

Comment:
Give to the searchView the same position of the app to keep the coherence with the runtime data